patches/net/minecraft/world/item/AxeItem.java.patch

```diff
@@ -42,20 +43,20 @@
                  if (optional2.isPresent()) {
                      p_308922_.playSound(p_309192_, p_308899_, SoundEvents.AXE_WAX_OFF, SoundSource.BLOCKS, 1.0F, 1.0F);
                      p_308922_.levelEvent(p_309192_, 3004, p_308899_, 0);
-@@ -99,8 +_,19 @@
+@@ -99,8 +_,18 @@
          }
      }
  
 +    public static BlockState getAxeStrippingState(BlockState originalState) {
 +        Block block = originalState.getBlock().builtInRegistryHolder().getData(net.neoforged.neoforge.registries.datamaps.builtin.NeoForgeDataMaps.STRIPPABLES).strippedBlock();
-+        return block.defaultBlockState().setValue(RotatedPillarBlock.AXIS, originalState.getValue(RotatedPillarBlock.AXIS));
++        return block.defaultBlockState();
```

Comment:
Isn't there a blockstate method that copies all matching properties from the source to the definition? Might be worth using that here.

Reply:
can't remember that off the top of my head